src/components/GoalDependencyList.tsx

```diff
@@ -22,6 +23,7 @@ interface GoalDependencyListByKindProps {
 }
 
 export const GoalDependencyListByKind = ({ id, goals = [], onClick, onRemove }: GoalDependencyListByKindProps) => {
+    const { themeId } = usePageContext();
```

Comment:
Why not to do this in GoalBadge?

Reply:
One-time call and N-times passing primitive value in prop versus N-times call function that read context

Reply:
👌